bayesian_model/processor.py

```diff
@@ -0,0 +1,145 @@
+            get_tagging_from_krnnt(text, path_to_processed_train_files + os.path.sep + KRNNT_FOLDER + os.path.sep +
+                                   training_data_file.split(".")[:-1][0] + ".xml")
+
+            convert_krnnt_output_file_to_xml(path_to_processed_train_files + os.path.sep + KRNNT_FOLDER + os.path.sep +
+                                             training_data_file.split(".")[:-1][0] + ".xml",
+                                             path_to_processed_train_files + os.path.sep + KRNNT_FOLDER + os.path.sep +
+                                             training_data_file.split(".")[:-1][0] + ".xml"
+                                             )
+
+            get_tagging_from_morphoDita(text,
+                                        path_to_processed_train_files + os.path.sep + MORPHODITA_FOLDER + os.path.sep +
+                                        training_data_file.split(".")[:-1][0] + ".xml")
+            get_tagging_from_wcrft2(text, path_to_processed_train_files + os.path.sep + WCRFT2_FOLDER + os.path.sep +
+                                    training_data_file.split(".")[:-1][0] + ".xml")
+
+
+def process_testing_data_to_xmls(path_to_data: str):
+    test_data_files = os.listdir(path_to_data + os.path.sep + 'test')
+    path_to_processed_test_files = PATH_TO_RAW_FILES + os.path.sep + "processed_test"
+
+    for test_data_file in test_data_files:
+        if test_data_file.endswith(".txt") and not os.path.isfile(
+                path_to_processed_test_files + os.path.sep + WCRFT2_FOLDER + os.path.sep +
+                test_data_file.split(".")[:-1][0] + ".xml"):
+            text = open(PATH_TO_RAW_FILES + os.path.sep + "test" + os.path.sep + test_data_file, "r",
+                        encoding="utf-8").read()
+            get_tagging_from_krnnt(text, path_to_processed_test_files + os.path.sep + KRNNT_FOLDER + os.path.sep +
+                                   test_data_file.split(".")[:-1][0] + ".xml")
+
+            convert_krnnt_output_file_to_xml(path_to_processed_test_files + os.path.sep + KRNNT_FOLDER + os.path.sep +
+                                             test_data_file.split(".")[:-1][0] + ".xml",
+                                             path_to_processed_test_files + os.path.sep + KRNNT_FOLDER + os.path.sep +
+                                             test_data_file.split(".")[:-1][0] + ".xml"
+                                             )
+
+            get_tagging_from_morphoDita(text,
+                                        path_to_processed_test_files + os.path.sep + MORPHODITA_FOLDER + os.path.sep +
+                                        test_data_file.split(".")[:-1][0] + ".xml")
+            get_tagging_from_wcrft2(text, path_to_processed_test_files + os.path.sep + WCRFT2_FOLDER + os.path.sep +
+                                    test_data_file.split(".")[:-1][0] + ".xml")
+
+
+def read_training_data(path_to_data: str, tagger_folder: str):
```

Comment:
Same here ;)
```suggestion
def read_data(path_to_data: str, tagger_folder: str, dataset_part: str):
```

Reply:
https://github.com/Ceglowa/ajp_1_dratwinski_gramacki/pull/4/commits/e4191758dd15ad239eb980cf79a4b1b567953baf

Reply:
Commited in https://github.com/Ceglowa/ajp_1_dratwinski_gramacki/pull/4/commits/e4191758dd15ad239eb980cf79a4b1b567953baf